src/py/aspen/database/models/tests/test_sequences.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+
+from ..physical_sample import PhysicalSample
+from ..sequences import (
+    SequencingInstrumentType,
+    SequencingProtocolType,
+    SequencingReads,
+    UploadedPathogenGenome,
+)
+from ..usergroup import Group
```

Comment:
i personally don't like using the `..` i think we should replace with the package name when we start using that syntax

Reply:
I generally agree with you, but I like it for intra-package imports.  I consider tests for a package an intra-package import.

Reply:
why do you like it for intra-package imports? Does it seem less redundant?

Reply:
If I refactor, it's less stuff that changes.